lib/index.js

```diff
@@ -24,9 +23,7 @@ var PouchMirror = module.exports = function (localDB, remote) {
   // Clone all the functions from the localDB into the PouchMirror instance

```

Comment:
Why not use this exact same strategy to clone your Strategy's API, instead of having to type everything out by hand?